internal/xds/translator/testdata/in/xds-ir/retry.yaml

```diff
@@ -0,0 +1,32 @@
+http:
+- name: "first-listener"
+  address: "0.0.0.0"
+  port: 10080
+  hostnames:
+  - "*"
+  path:
+    mergeSlashes: true
+    escapedSlashesAction: UnescapeAndRedirect
+  routes:
+  - name: "first-route"
+    hostname: "*"
+    retry:
```

Comment:
can we add another case with a empty `retry: {}` to its easier to cross check defaults in the future, thanks